compiler/src/parsing/well_formedness.re

```diff
@@ -153,7 +156,7 @@ let types_have_correct_case = (errs, super) => {
 let modules_have_correct_case = (errs, super) => {
   let check_uppercase = (loc, s) => {
     let first_char = s.[0];
-    if (first_char != BatChar.uppercase(first_char)) {
+    if (Char_utils.is_lowercase_letter(first_char)) {
```

Comment:
Same deal here.

Reply:
What's the same?

Reply:
That this logic should be "isn't uppercase" rather than "is lowercase."

Reply:
Is that just because special characters should throw here?

Reply:
Updated.